mu-plugins/blocks/local-navigation-bar/src/view.js

```diff
@@ -36,6 +36,64 @@ function init() {
 
 		document.addEventListener( 'scroll', debounce( onScroll ), { passive: true } );
 		onScroll();
+
+		// Check the size of child elements to determine if the local navigation
+		// menu should be collapsed in mobile-view by default. If so, toggle a
+		// CSS class to show the nav block with {"overlayMenu":"always"}
+		// added by `add_extra_navigation`.
+		const onResize = () => {
+			// Bail early on small screens, the visible nav block is already mobile.
+			if ( window.innerWidth <= 600 ) {
+				container.classList.remove( 'wporg-show-collapsed-nav' );
+				return;
+			}
+
+			// Fetch the navWidth from a data value which is set on page load,
+			// so that the uncollapsed visible menu's width is used.
+			let navWidth = container.dataset.navWidth;
+			if ( ! navWidth ) {
+				const navElement = container.querySelector( 'nav:not(.wporg-is-collapsed-nav)' );
+				const navGap = ( window.getComputedStyle( navElement ).gap || '20px' ).replace( 'px', '' ) * 1;
+				// Get the nav width based on items, so that it stays
+				// consistent even if the menu wraps to a new line.
+				const menuItems = navElement.querySelectorAll( '.wp-block-navigation__container > li' );
+				navWidth =
+					[ ...menuItems ].reduce(
+						( acc, current ) => ( acc += current.getBoundingClientRect().width ),
+						0
+					) +
+					navGap * ( menuItems.length - 1 ); // 20px gap between items.
+
+				// Save the value for future resize callbacks.
+				container.dataset.navWidth = Math.ceil( navWidth );
+			}
+
+			const {
+				paddingInlineStart = '0px',
+				paddingInlineEnd = '0px',
+				gap = '0px',
+			} = window.getComputedStyle( container );
+
+			const availableWidth =
+				window.innerWidth -
+				paddingInlineStart.replace( 'px', '' ) * 1 -
```

Comment:
I have always used `parseInt` for this type of conversion, not sure what's better, but maybe it's easier to understand the intention
```suggestion
				parseInt(paddingInlineStart, 10) -
```

Reply:
Personally I prefer `Number` over `parseInt` (the 10 always throws me off 🤷🏻), but this is a good point, I'll update it.

Reply:
Ah, `parseInt` parses up to a non-number value, so I wouldn't need to remove the `'px'`… Okay, that's probably a good reason to use that here :)

Reply:
Done in https://github.com/WordPress/wporg-mu-plugins/pull/573/commits/8e5458bd30923943a44636fefc5d1daa145090ed